lib/src/rules/dart_imports_go_first.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library linter.src.rules.dart_imports_go_first;
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:linter/src/analyzer.dart';
+import 'package:linter/src/util/dart_type_utilities.dart';
+
+const desc = r"Place '*dart:*' imports before other imports.";
```

Comment:
The description is used as the error message, displayed in the IDE and on the console. In neither case will markdown be processed, so it would be better to remove the asterisks.